semeio/jobs/rft/trajectory.py

```diff
@@ -65,11 +68,34 @@ def get_pressure(self):
             return self.pressure
         return -1
 
-    def update_pressure_from_rft(self, rft):
+    def get_swat(self):
+        if self.is_active():
+            return self.swat
+        return -1
+
+    def get_sgas(self):
+        if self.is_active():
+            return self.sgas
+        return -1
+
+    def get_soil(self):
+        if self.is_active():
+            return self.soil
+        return -1
```

Comment:
Wont rule out that I am blind, but are these used anywhere?

Reply:
Not currently in semeio, was just added so the API was analogous as `get_pressure()` (which is used once in semeio, in a test). There are user scripts around that might end up using this API, unless we can make then redundant.

Reply:
Removed. We should redirect users to the CSV output.